examples/ImGuiModule.cmake

```diff
@@ -0,0 +1,312 @@
+if(NOT DEFINED ImGuiModule_CMAKE_INCLUDED)
+    set(ImGuiModule_CMAKE_INCLUDED 1)
+else()
+    return()
+endif()
```

Comment:
There's no need for this. CMake will error over a number of things. No need to hold CMake maintainers' hands IMO. Include guards are not a common thing in CMake configurations.